### PR TITLE
🟥 Update to latest 3.x redis client

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/mailroom/queue.py [hunk 1]
```diff
@@ -231,10 +231,10 @@ def _queue_task(pipe, org_id, queue, task_type, task, priority):
     active_queue = ACTIVE_PATTERN % queue
 
     # push onto our org queue
-    pipe.zadd(org_queue, score, json.dumps(payload))
+    pipe.zadd(org_queue, {json.dumps(payload): score})
 
     # and mark that org as active
-    pipe.zincrby(active_queue, org_id, 0)
+    pipe.zincrby(active_queue, 0, org_id)
```
Comment (rowanseymour) [inline temba/mailroom/queue.py:237]: https://github.com/andymccurdy/redis-py/blob/1a41cfd95a53a95b078084d8627be6b6fba3bb71/CHANGES#L271-L272

File: temba/mailroom/queue.py [hunk 2]
```diff
@@ -231,10 +231,10 @@ def _queue_task(pipe, org_id, queue, task_type, task, priority):
     active_queue = ACTIVE_PATTERN % queue
 
     # push onto our org queue
-    pipe.zadd(org_queue, score, json.dumps(payload))
+    pipe.zadd(org_queue, {json.dumps(payload): score})
```
Comment (rowanseymour) [inline temba/mailroom/queue.py:234]: https://github.com/andymccurdy/redis-py/blob/1a41cfd95a53a95b078084d8627be6b6fba3bb71/CHANGES#L278-L280

File: pip-requires.txt [hunk 3]
```diff
@@ -50,7 +50,7 @@ pycrypto
 python-dateutil==2.2
 pytz
 psycopg2-binary
-redis<3.0  # see https://github.com/celery/celery/issues/4495
```
Comment (rowanseymour) [inline pip-requires.txt:53]: fixed long ago

File: pip-freeze.txt [hunk 4]
```diff
@@ -63,7 +63,7 @@ django-hamlpy==1.3
     # via -r pip-requires.txt
 django-mptt==0.8.7
     # via -r pip-requires.txt
-django-redis==4.10.0
```
Comment (rowanseymour) [inline pip-freeze.txt:66]: this was the last version to support redis<3

File: temba/utils/cache.py [hunk 5]
```diff
@@ -18,7 +18,7 @@ def get_cacheable(cache_key, callable, r=None, force_dirty=False):
             return json.loads(force_text(cached))
 
     (calculated, cache_ttl) = callable()
-    r.set(cache_key, json.dumps(calculated), cache_ttl)
+    r.set(cache_key, json.dumps(calculated), ex=cache_ttl or None)
```
Comment (rowanseymour) [inline temba/utils/cache.py:21]: old client ignored `ex` if it was falsey, new client only ignores `None`

**Review by norkans7 — APPROVED** [review]

**Review by nicpottier — APPROVED** [review]
Lets definitely give this a heavy pass on staging, I remember the last client update they changed the parameter ordering on a few calls that caused some mayhem.

**Comment by rowanseymour** [comment]
For sure - the good news is that we now do very little redis'ing in RP

**Pull request merged**